internal/services/privatedns/private_dns_zone_virtual_network_link_resource_test.go

```diff
@@ -109,11 +109,6 @@ resource "azurerm_virtual_network" "test" {
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
   address_space       = ["10.0.0.0/16"]
-
-  subnet {
-    name           = "subnet1"
-    address_prefix = "10.0.1.0/24"
-  }
```

Comment:
why are we removing this from the tests?

Reply:
@katbyte Deleted because of test failure.

Reply:
what test error was it fixing? what changed that is no longer required?

Reply:
@katbyte The tests have been modified, please check again, thanks.

Reply:
@katbyte I tried it without modifying tests and it works.